### ISSUE TITLE
Bios and Roms XML manifests

### ISSUE THREAD

**Issue by sveetch** [recalbox/recalbox-os#357]
Would be more usable for third apps or another code to have a clean Manifest for supported bios and rom systems.

I got this idea when working on recalbox-webconfig and my attempt to do a recalbox web manager in Django, where we need to parse readme.txt files that is such a pain and probably not very stable (since readme does not have a real stable syntax).

So i will compile datas from the readme files to produce Manifests XML files, so we can parse them correctly. I will ship them with some little Python script to automatically rebuild the readme files, so you dont have to manually rewrite them again each time.

**Comment by sveetch** [comment]
So here is my XML file : https://gist.github.com/sveetch/a5a25ce212b8b063c9f2

I created it using infos from readme files in rom systems directories.

I collected the system keyname, the system name, supported rom file extensions and download links.

I saw that some readme contains comments and i didnt collected them because they are in english and so cant be used to build the "lisezmoi.txt" file. This is a problem because i planned to build the readme file from the XML with an unique template for each language, but with the comments i think we would have to maintain different templates for some systems.

Do you mind about dropping the "lisezmoi.txt" files and keeping only the readme in english ?

I'm waiting for your answers before to continue to code the readme builder using the manifest, so you can integrate it within the os building.

**Comment by sveetch** [comment]
Oh yeah and XML is verbose, if you want i can't build a JSON manifest instead

**Comment by digitalLumberjack** [comment]
Yep i like this, we should add the bios informations. Each bios has a name and a checksum.

**Comment by sveetch** [comment]
That's nice because i allready added the bios infos yesterday (look the gist) :)

**Comment by digitalLumberjack** [comment]
Cool ! It misses the lynx bios, that is also missing in the original readme.

**Comment by sveetch** [comment]
Ok the Lynx bios has been added

**Comment by sveetch** [comment]
Added the Atari 7800 bios file. The manifest should be sync with the 3.3.0 beta now

**Comment by sveetch** [comment]
Ok i stated with @digitalLumberjack on requirements to deliver the command line script to build rom and bios readme.

* Recalbox drop support for french readme "lisezmoi.txt" so we won't have to manage multiple templates for specifies throught two languages;
* Add new elements to XML to be able to have some custom comments to reproduce rom readme files diversity;
* Develop the command line script and it's template(s) into a new repository named "recalbox-manifesto", so the recalbox devs can integrate it as a git submodule;

Will come back here when it's ready

**Comment by sveetch** [comment]
Ok it has been done on :

  https://github.com/sveetch/recalbox-manifesto

The Readme should contain everything you need to use it. Let me know if it's ok and ready for you, use the github issues if needed.

**Comment by digitalLumberjack** [comment]
okay :)

**Comment by digitalLumberjack** [comment]
I though it was the integration of the manifesto in webmanager :P
